ROMFS/px4fmu_common/init.d/airframes/13014_vtol_babyshark

```diff
@@ -96,9 +96,45 @@ param set-default VT_PSHER_RMP_DT 2
 param set-default VT_TRANS_MIN_TM 4
 param set-default VT_TYPE 2
 
-set MIXER babyshark
-set MIXER_AUX pass
 
-# Mark outputs for the alternate rate
-# or D-Shot
-set PWM_OUT 5678
+param set-default CA_AIRFRAME 2
+param set-default CA_ROTOR_COUNT 5
+param set-default CA_ROTOR0_PX 1
+param set-default CA_ROTOR0_PY 1
+param set-default CA_ROTOR1_PX -1
+param set-default CA_ROTOR1_PY -1
+param set-default CA_ROTOR2_PX 1
+param set-default CA_ROTOR2_PY -1
+param set-default CA_ROTOR2_KM -0.05
+param set-default CA_ROTOR3_PX -1
+param set-default CA_ROTOR3_PY 1
+param set-default CA_ROTOR3_KM -0.05
+param set-default CA_ROTOR4_AX 1.0
+param set-default CA_ROTOR4_AZ 0.0
+
+param set-default CA_SV_CS_COUNT 3
+param set-default CA_SV_CS0_TYPE 1
+param set-default CA_SV_CS0_TRQ_R -0.5
```

Comment:
```suggestion
param set-default CA_SV_CS0_TRQ_R -1
```

Reply:
as we here only have a single aileron defined the effectiveness of it has to be 1.

Reply:
addressed in https://github.com/PX4/PX4-Autopilot/pull/19990/commits/780a39386c52bb297d56dc2b3e46737cf99ddb0c

Reply:
Thanks